test/common/stats/tag_extractor_impl_test.cc

```diff
@@ -273,14 +273,15 @@ TEST(TagExtractorTest, DefaultTagExtractors) {
 
   Tag grpc_service;
   grpc_service.name_ = tag_names.GRPC_BRIDGE_SERVICE;
-  grpc_service.value_ = "grpc_service_1";
+  grpc_service.value_ = "fully.qualified.grpc_service_1";
 
   Tag grpc_method;
   grpc_method.name_ = tag_names.GRPC_BRIDGE_METHOD;
   grpc_method.value_ = "grpc_method_1";
 
-  regex_tester.testRegex("cluster.grpc_cluster.grpc.grpc_service_1.grpc_method_1.success",
-                         "cluster.grpc.success", {grpc_cluster, grpc_method, grpc_service});
+  regex_tester.testRegex(
+      "cluster.grpc_cluster.grpc.fully.qualified.grpc_service_1.grpc_method_1.success",
+      "cluster.grpc.success", {grpc_cluster, grpc_method, grpc_service});
```

Comment:
I don't understand how ".success" was preserved after matching with GRPC_BRIDGE_SERVICE, can you explain what is going on?

Reply:
No, I didn't know that.  Makes more sense now.

Reply:
You have changed the existing test. Can you restore that and also add your new test?